src/app/shell/personal-cabinet/personal-cabinet.component.html

```diff
@@ -7,7 +7,7 @@ <h1 class="title">{{userRole === 'provider' ? 'КАБІНЕТ НАДАВАЧА'
       {{ roles[userRole] | uppercase }}</a>
     <a *ngIf="userRole !== Role.techAdmin" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./workshops'">{{ userRole === Role.parent ? 'МОЇ ГУРТКИ' : 'ГУРТКИ' }}</a>
     <a *ngIf="userRole !== Role.techAdmin" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./applications'">ЗАЯВИ</a>
-    <a *ngIf="userRole == Role.provider" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./users'">КОРИСТУВАЧІ</a>
+    <a *ngIf="userRole == Role.provider" mat-tab-link [routerLinkActive]="['active']" [routerLink]="'./users'">АДМІНІСТРУВАННЯ</a>
```

Comment:
I don't think it is a good link naming, if we have changed users to administration, then I believe we should change the route as well from "users" to "administration" or something like that
![image](https://user-images.githubusercontent.com/27493959/158139141-e63541a8-1e02-4244-a403-19e6a95638fc.png)